src/Workspaces/CSharp/Portable/Rename/CSharpRenameRewriterLanguageService.cs

```diff
@@ -639,22 +639,23 @@ private SyntaxTrivia RenameInCommentTrivia(SyntaxTrivia trivia)
 
             private SyntaxToken RenameWithinToken(SyntaxToken oldToken, SyntaxToken newToken)
             {
+                ImmutableSortedSet<TextSpan> subSpansToReplace = null;
```

Comment:
can you just assign this an empty set?

Reply:
The null value has a specific meaning here. I have added doc comments to the field as well as comments inside `ReplaceMatchingSubStrings` where this set is eventually used.